### ISSUE TITLE
Error when installing from pip/building

### ISSUE THREAD

**Issue by f1refa11** [lemire/fastrand#19]
Error:
```
Collecting fastrand
  Downloading https://files.pythonhosted.org/packages/4a/c9/84ae3fa10cb24bd5ac88845a55ddddc572d165218e29613063763f12d8c4/fastrand-1.7.0.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-lr57d0c2/fastrand/setup.py", line 9, in <module>
        long_description=open('README.md', 'r').read(),
      File "/usr/lib/python3.5/encodings/ascii.py", line 26, in decode
        return codecs.ascii_decode(input, self.errors)[0]
    UnicodeDecodeError: 'ascii' codec can't decode byte 0xe2 in position 124: ordinal not in range(128)
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-lr57d0c2/fastrand/
```
Any way to fix this?

**Comment by f1refa11** [comment]
Tried installing version 1.2.4, it worked👍. Maybe i find other newer versions which do not have this error.

**Comment by lemire** [comment]
Thanks for the report. It should be fixed in version 1.7.1.

Please open an new issue if that's not the case.
